tests/unit/expected_config/config.conf

```diff
@@ -1,13 +1,6 @@
 configuration:
   nrfUri: http://nrf:8081
   webuiUri: sdcore-webui:9876
-  nsiList:
-  - nsiInformationList:
-    - nrfId: http://nrf:8081/nnrf-nfm/v1/nf-instances
-      nsiId: 22
-    snssai:
-      sd: "010203"
-      sst: 1
```

Comment:
Doesn't this PR conflict with your own #370

Reply:
Nope, this one is for v1.5.
I assumed we want to have this squared up in the released version as well.